app/services/project_region_migrator.rb

```diff
@@ -0,0 +1,12 @@
+class ProjectRegionMigrator
+  def initialize(project)
+    @project = project
+  end
+
+  def migrate_up!
+    if @project.region.nil?
+      establishment = @project.establishment
+      @project.update!(region: establishment.region_code)
```

Comment:
I would be tempted to use `save(validate: false)` on this, we've been stung before where a project is not valid so the data migration fails, I feel like we know this 'region' is correct so we can skip validation as part of this - be intereseted in your thoughts?